config/405-github-ok-to-test-comment.yaml

```diff
@@ -26,7 +26,7 @@ spec:
     - name: action
       value: "$(body.action)"
     - name: trigger_target
-      value: "ok-to-test-comment"
+      value: pull_request
```

Comment:
does all of them need to be `pull_request` ?
what does trigger target refer to ?

Reply:
So I explained a bit more in the comments here : 

https://github.com/openshift-pipelines/pipelines-as-code/pull/442/files#diff-6d6547a78b2a5d6fccaf1bf7f60f8bf5eb68b3cf63725d2e5b644b5861c577c7R6

On gitlab, events are called "Merge Request hooks" or "Note Requests hooks" but comments usually targets a merge request, in paac we use the word "pull_request" to differentiate between just a push (where we are not going to report stuff in a comment) and pull request (where we can put a comment)....